tools/uswds-boilerplate-on-codepen.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: USWDS Boilerplate on CodePen
+nav: Tools
+permalink: /tools/uswds-boilerplate-on-codepen/
+sidenav: tools-side-nav
+---
+# {{ page.title }}
+
+CodePen is an web-based editor for experimenting with and prototyping HTML, CSS, and JavaScript. It's the easiest way to start getting familiar with USWDS code snippets and design tokens because you don't have to worry about dev environments and hosting. All you do is put some code in the releavnt boxes and it just works.
+
+## Getting started
+
+1. You can edit CodePens without an account, but since you'll probably want to save your work, we recommend you [sign up for a free account](https://codepen.io/accounts/signup/user/free).
+1. Open our [USWDS Boilerplate pen](https://codepen.io/pglevy/pen/abBgJbe).
```

Comment:
Is there anyway to have these links open up in a new tab/window?

Reply:
Not easily in the code the way GH Pages works now. Command-click? :) I will look into using another markdown renderer, but not sure if we want to make that change just yet.